pkg/scheduler/plugins/rescheduling/node_utilization_util_test.go

```diff
@@ -0,0 +1,441 @@
+/*
```

Comment:
There are some pod related utils in ``pkg/scheduler/util/test_util.go`, we can reuse them: )

Reply:
Got it